application/src/pages/Cart/EmptyCart/EmptyCart.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { EmptyCart } from './EmptyCart';
+import { store } from '../../../bll/store';
+import i18n from '../../../utils/i18next';
+
+describe('EmptyCart', () => {
+  beforeEach(() => {
+    i18n.init();
```

Comment:
You can do it without beforeEach in this case

Reply:
removed